monitor/census.go

```diff
@@ -751,7 +754,9 @@ func TranscodeTry(nonce, seqNo uint64) {
 		} else {
 			av.tries[seqNo] = tryData{tries: 1, first: time.Now()}
 		}
-		glog.Infof("Trying to transcode segment nonce=%d seqNo=%d try=%d", nonce, seqNo, try)
+		if glog.V(logLevel) {
+			glog.Infof("Trying to transcode segment nonce=%d seqNo=%d try=%d", nonce, seqNo, try)
```

Comment:
Do you think this would be cleaner as `glog.V(loglevel).Info("foo")`? Would save having to write a lot of `if` statements.

Reply:
Yeah, that's the convention we use elsewhere. Presumably @darkdarkdragon  was trying to save a few cycles since this form doesn't evaluate the arguments to `Infof` , but I agree it's very verbose.

I'm indifferent about `census.go` but would rather not see this pattern proliferate to the rest of the codebase.

Reply:
@iameli @j0sh removed `if V()` pattern